Wrappers/Python/cil/utilities/callbacks.py

```diff
@@ -0,0 +1,85 @@
+
+from abc import ABC, abstractmethod
+
+from tqdm.auto import tqdm as tqdm_auto
+from tqdm.std import tqdm as tqdm_std
+
+
+class Callback(ABC):
+    '''Base Callback to inherit from for use in :code:`Algorithm.run(callbacks: list[Callback])`.'''
```

Comment:
The list of parameters should go in the class description rather than in in the init